great_expectations/render/renderer/content_block/validation_results_table_content_block.py

```diff
@@ -156,13 +156,13 @@ def _get_unexpected_table(cls, evr):
             sampled_values_set = set()
             for unexpected_value in result.get("partial_unexpected_list"):
                 if unexpected_value:
-                    string_unexpected_value = unexpected_value
+                    string_unexpected_value = str(unexpected_value)
                 elif unexpected_value == "":
                     string_unexpected_value = "EMPTY"
                 else:
                     string_unexpected_value = "null"
                 if string_unexpected_value not in sampled_values_set:
-                    table_rows.append([string_unexpected_value])
+                    table_rows.append([unexpected_value])
```

Comment:
Do table rows now how to render dicts?

Reply:
Answer: by placing the raw dict here, we are punting to the view engine, where the `render_content_block` method will return the dict as is (unless it contains a "content_block_type" key used to determine which one content block renderer should be used.